confd/pkg/resource/template/template_funcs.go

```diff
@@ -111,12 +111,35 @@ var (
 	}
 )
 
-func filterMatchCIDR(cidr string, operator v3.BGPFilterMatchOperator) (string, error) {
+func filterMatchPrefixLength(cidr string, prefixLength v3.BGPFilterPrefixLength) string {
+	cidrIP, cidrNet, _ := net.ParseCIDR(cidr)
```

Comment:
I think it would be worth at least including a warning log for the error here instead of ignoring it entirely. 

I know we _shouldn't_ hit this since the CIDR should be pre-validated elsewhere, but best to play it safe.

Reply:
Added a print here, not sure if we want to bail out and only return `cidr` with no prefixLength, error, etc